src/cli/keepassxc-cli.cpp

```diff
@@ -32,6 +32,7 @@
 #include "Utils.h"
 #include "config-keepassx.h"
 #include "core/Bootstrap.h"
+#include "core/Metadata.h"
```

Comment:
any particular reason for this include?

Reply:
I'm not 100% sure to be honest. My guess is that since `Bootstrap` was previously including `gui/MainWindow.h`, and that `MainWindow` ends up importing pretty much every core module, we were able to not include here, but now we have to.